README.md

```diff
@@ -20,128 +30,21 @@ Below is the list of items that are currently synchronized:
-![image](https://user-images.githubusercontent.com/6945229/155609826-d3938767-6287-4626-94a3-aea4fd758204.png)
-![image](https://user-images.githubusercontent.com/6945229/155610226-799c79de-719b-44af-9a07-2aaabfea5510.png)
-
-
-## Installation
-
-The plugin is available as a Python package in pypi and can be installed with pip
-
-```shell
-pip install nautobot-ssot-aci
-```
-
-> The plugin is compatible with Nautobot 1.2.0 and higher
-
-To ensure Nautobot SSoT for Cisco ACI is automatically re-installed during future upgrades, create a file named `local_requirements.txt` (if not already existing) in the Nautobot root directory (alongside `requirements.txt`) and list the `nautobot-ssot-aci` package:
-
-```no-highlight
-# echo nautobot-ssot-aci >> local_requirements.txt
-```
-
-Once installed, the plugin needs to be enabled in your `nautobot_config.py`
-
-```python
-# In your nautobot_config.py
-PLUGINS = ["nautobot_ssot_aci"]
-```
-
-In addition, the plugin behavior can be controlled with the following list of settings.
-```python
-PLUGINS_CONFIG = {
-    "nautobot_ssot": {
-        "hide_example_jobs": True,
-    },
-    "nautobot_ssot_aci": {
-        # URL and credentials should be configured as environment variables on the host system
-        "apics": {x: os.environ[x] for x in os.environ if "APIC" in x},
-        # Tag which will be created and applied to all synchronized objects.
-        "tag": "ACI",
-        "tag_color": "0047AB",
-        # Tags indicating state applied to synchronized interfaces.
-        "tag_up": "UP",
-        "tag_up_color": "008000",
-        "tag_down": "DOWN",
-        "tag_down_color": "FF3333",
-        # Manufacturer name. Specify existing, or a new one with this name will be created.
-        "manufacturer_name": "Cisco",
-        # Exclude any tenants you would not like to bring over from ACI.
-        "ignore_tenants": ["common", "mgmt", "infra"],
-        # The below value will appear in the Comments field on objects created in Nautobot
-        "comments": "Created by ACI SSoT Plugin",
-    }
-```
 
-Some of the above settings can be omitted, and if omitted the below defaults will take effect:
-```python
-default_settings = {"tag": "ACI",
-                   "tag_color": "FF3333",
-                   "manufacturer_name": "Cisco",
-                   }                    
-```
-The APIC URL and credentials should be created as environment variables on the host system, for example:
+## Documentation
 
-```bash
-export NAUTOBOT_APIC_BASE_URI_NTC=https://aci.cloud.networktocode.com
-export NAUTOBOT_APIC_USERNAME_NTC=admin
-export NAUTOBOT_APIC_PASSWORD_NTC=not_so_secret_password
-export NAUTOBOT_APIC_VERIFY_NTC=False
-export NAUTOBOT_APIC_SITE_NTC="NTC ACI"
-export NAUTOBOT_APIC_TENANT_PREFIX_NTC="NTC_ACI"
-```
+Documentation is hosted on ReadTheDocs at [Nautobot Single Source of Truth Documentation](https://nautobot-plugin-ssot-aci.readthedocs.io/).
```

Comment:
My bad, change 

"[Nautobot Single Source of Truth Documentation]"

to

"[Nautobot SSoT for Cisco ACI Documentation]"

Reply:
Good catch, I've added this to the latest commit